jerry-core/config.h

```diff
@@ -119,6 +119,10 @@
 # define JERRY_BUILTIN_WEAKMAP JERRY_ESNEXT
 #endif /* !defined (JERRY_BUILTIN_WEAKMAP) */
 
+#ifndef JERRY_BUILTIN_WEAKREF
+# define JERRY_BUILTIN_WEAKREF JERRY_ESNEXT
```

Comment:
I also rise a question about this, as the native api such as (iotjs/napi) may need weakref to finalize localy things, and iotjs need support both es5.1 and esnext, may us enable JERRY_BUILTIN_WEAKREF for either es5.1 or es.next by default? such as 
```

/**
 * ES2015+ related features, but enabled by
 * JERRY_BUILTINS or JERRY_ESNEXT if not defined
 */
#if JERRY_BUILTINS || JERRY_ESNEXT
# define JERRY_BUILTIN_OR_ESNEXT 1
#else
# define JERRY_BUILTIN_OR_ESNEXT 0
#endif

#ifndef JERRY_BUILTIN_WEAKREF
# define JERRY_BUILTIN_WEAKREF JERRY_BUILTIN_OR_ESNEXT
#endif /* !defined (JERRY_BUILTIN_WEAKREF) */

```

Reply:
> I also rise a question about this, as the native api such as (iotjs/napi) may need weakref to finalize localy things, and iotjs need support both es5.1 and esnext, may us enable JERRY_BUILTIN_WEAKREF for either es5.1 or es.next by default? such as
> 

I don't agree with this suggestion, similar to globalThis:
https://github.com/jerryscript-project/jerryscript/pull/4575#issuecomment-775004709